README.md

```diff
@@ -66,6 +66,7 @@ On the host via the script
       -i <TEXT>  [optional] Specify image ID used within Sysdig (ex: -i '<64 hex characters>')
       -d <PATH>  [optional] Specify image digest (ex: -d 'sha256:<64 hex characters>')
       -m <PATH>  [optional] Path to Docker image manifest (ex: -m ./manifest.json)
+      -l  [optional] To invoke curl commands as locked connections
```

Comment:
@haresh-suresh – To clarify, I don't think "locked" describes anything. Should say something like "Enable HTTP basic authentication".

Reply:
I agree... I am confused by the "locked" meaning.